AEMaaCS image build fails due to Core Forms Components artifacts

Reported error:
```
[91mjava.lang.RuntimeException: java.lang.IllegalStateException: Artifact override rule required to select between these two artifacts Artifact [id=com.adobe.aem:core-forms-components-apps:zip:cloud:1.0.16] and Artifact [id=adobe.cq60:core-forms-components-apps:zip:cp2fm-converted:1.0.14]. The rule must be specified for [adobe.cq60:core-forms-components-apps:zip:cp2fm-converted:1.0.16]
[0m[91m	at com.adobe.granite.fact.command.BuildImageCommand.lambda$doMergeFeatures$25(BuildImageCommand.java:768)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1632)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.MatchOps$MatchTask.doLeaf(MatchOps.java:306)
	at java.base/java.util.stream.MatchOps$MatchTask.doLeaf(MatchOps.java:277)
	at java.base/java.util.stream.AbstractShortCircuitTask.compute(AbstractShortCircuitTask.java:115)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinTask.doInvoke(ForkJoinTask.java:408)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:736)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateParallel(MatchOps.java:242)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateParallel(MatchOps.java:196)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.base/java.util.stream.ReferencePipeline.allMatch(ReferencePipeline.java:533)
	at com.adobe.granite.fact.command.BuildImageCommand.doMergeFeatures(BuildImageCommand.java:770)
	at com.adobe.granite.fact.command.BuildImageCommand$2.run(BuildImageCommand.java:1000)
	at java.base/java.lang.Thread.run(Thread.java:834)
[0m[91mCaused by: java.lang.IllegalStateException: Artifact override rule required to select between these two artifacts Artifact [id=com.adobe.aem:core-forms-components-apps:zip:cloud:1.0.16] and Artifact [id=adobe.cq60:core-forms-components-apps:zip:cp2fm-converted:1.0.14]. The rule must be specified for [adobe.cq60:core-forms-components-apps:zip:cp2fm-converted:1.0.16]
	at org.apache.sling.feature.builder.BuilderUtil.selectArtifactOverride(BuilderUtil.java:267)
	at org.apache.sling.feature.builder.BuilderUtil.mergeArtifacts(BuilderUtil.java:158)
	at org.apache.sling.feature.builder.BuilderUtil.mergeExtensions(BuilderUtil.java:649)
	at org.apache.sling.feature.builder.BuilderUtil.mergeExtensions(BuilderUtil.java:683)
	at org.apache.sling.feature.builder.FeatureBuilder.merge(FeatureBuilder.java:391)
	at org.apache.sling.feature.builder.FeatureBuilder.assemble(FeatureBuilder.java:210)
	at com.adobe.granite.fact.command.AggregateFeaturesCommand.doCall(AggregateFeaturesCommand.java:108)
	at com.adobe.granite.fact.command.AbstractRepoCommand.call(AbstractRepoCommand.java:66)
	at com.adobe.granite.fact.command.BuildImageCommand.lambda$doMergeFeatures$25(BuildImageCommand.java:746)
	... 20 more
```

--- a/.github/workflows/maven-deploy-cm.yml
+++ b/.github/workflows/maven-deploy-cm.yml
@@ -121,7 +121,7 @@ jobs:
           counter=0
           max=${CM_TEST_RETRIES}
           echo "Looking for ${ARCHETYPE_COMMIT}: ${counter}"
-          until [[ $(curl --silent ${CM_TEST_URL} | grep ${ARCHETYPE_COMMIT}) ]]; do \
+          until [[ $(curl --silent ${CM_TEST_URL}?$(date +%s) | grep ${ARCHETYPE_COMMIT}) ]]; do \
             if [ ${counter} -eq ${max} ]; then \
               echo "Could not find ${ARCHETYPE_COMMIT} after ${counter} tries!"; \
               exit 1; \

--- a/src/main/archetype/all/pom.xml
+++ b/src/main/archetype/all/pom.xml
@@ -194,7 +194,7 @@
                             <target>/apps/${appId}-vendor-packages/application/install</target>
                         </embedded>
 #end
-#if ( $includeExamples == "y" and ( ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") and $aemVersion == "cloud" ) )
+#if ( $includeExamples == "y" and ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") )
                         <embedded>
                             <groupId>com.adobe.aem</groupId>
                             <artifactId>core-forms-components-examples-apps</artifactId>

--- a/src/main/archetype/all/pom.xml
+++ b/src/main/archetype/all/pom.xml
@@ -197,7 +197,13 @@
 #if ( $includeExamples == "y" and ( ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") and $aemVersion == "cloud" ) )
                         <embedded>
                             <groupId>com.adobe.aem</groupId>
-                            <artifactId>core-forms-components-examples-all</artifactId>
+                            <artifactId>core-forms-components-examples-apps</artifactId>
+                            <type>zip</type>
+                            <target>/apps/${appId}-vendor-packages/application/install</target>
+                        </embedded>
+                        <embedded>
+                            <groupId>com.adobe.aem</groupId>
+                            <artifactId>core-forms-components-examples-content</artifactId>
                             <type>zip</type>
                             <target>/apps/${appId}-vendor-packages/application/install</target>
                         </embedded>
@@ -425,7 +431,12 @@
 #if ( $includeExamples == "y" and ( ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") and $aemVersion == "cloud" ) )
         <dependency>
             <groupId>com.adobe.aem</groupId>
-            <artifactId>core-forms-components-examples-all</artifactId>
+            <artifactId>core-forms-components-examples-apps</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>com.adobe.aem</groupId>
+            <artifactId>core-forms-components-examples-content</artifactId>
             <type>zip</type>
         </dependency>
 #end

--- a/src/main/archetype/pom.xml
+++ b/src/main/archetype/pom.xml
@@ -895,7 +895,13 @@ Bundle-DocURL:
 #if ( $includeExamples == "y" and ( ($includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") and $aemVersion == "cloud" ) )
             <dependency>
                 <groupId>com.adobe.aem</groupId>
-                <artifactId>core-forms-components-examples-all</artifactId>
+                <artifactId>core-forms-components-examples-apps</artifactId>
+                <type>zip</type>
+                <version>${core.forms.components.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.adobe.aem</groupId>
+                <artifactId>core-forms-components-examples-content</artifactId>
                 <type>zip</type>
                 <version>${core.forms.components.version}</version>
             </dependency>

